src/cpp/ext/filters/census/client_filter.cc

```diff
@@ -152,6 +158,11 @@ void CensusClientCallData::Destroy(grpc_call_element* elem,
   const uint64_t request_size = GetOutgoingDataSize(final_info);
   const uint64_t response_size = GetIncomingDataSize(final_info);
   double latency_ms = absl::ToDoubleMilliseconds(absl::Now() - start_time_);
+  std::vector<std::pair<opencensus::tags::TagKey, std::string>> tags =
+      context_.tags().tags();
+  tags.emplace_back(ClientMethodTagKey(), method_);
```

Comment:
Does this need to be `std::string(method_)` to make [this](https://source.cloud.google.com/results/invocations/5cce2166-c286-4bd3-a916-a221974125cf/targets/grpc%2Fcore%2Fpull_request%2Flinux%2Fgrpc_bazel_build/log) pass?

Reply:
I am unsure how to run this test locally, it passes when I run it manually. So I expect its related to a different environment at compilation time. I tried the --use_docker flag as well, but wasn't able to reproduce the failure. Any suggestions?

tools/run_tests/run_tests.py -l c++ --use_docker

I've checked in a commit which compiles, using your std::string(method_) suggestion. Hopefully this will work. Though I am not sure how to trigger the tests to run again